Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -1225,7 +1301,11 @@ A number field `K` is totally complex if all of its infinite places are complex.
 @[mk_iff] class IsTotallyComplex (K : Type*) [Field K] [NumberField K] where
   isComplex : ∀ v : InfinitePlace K, v.IsComplex
 
-variable {K : Type*} [Field K] [NumberField K]
+variable {F : Type*} [Field F] [NumberField F] {K : Type*} [Field K] [NumberField K]
+
+theorem IsTotallyComplex.ofRingEquiv [IsTotallyComplex F] (f : F ≃+* K) :
```

Comment:
Is this needed for this PR? If not, I do not suggest to split it into another one, but perhaps add a comment in the PR description.

Reply:
I removed it. I'll add it in a later PR when I'll need it